docs/development/doc_guide.rst

```diff
@@ -1,76 +1,633 @@
-************************
-Documentation Guidelines
-************************
+*******************
+Documentation Guide
+*******************
 
-This document describes the documentation requirements and guidelines
-to be followed during the development of PlasmaPy and affiliated
-packages.
+Documentation that is up-to-date and understandable is vital to the
+health of a software project. This page describes the documentation
+requirements and guidelines to be followed during the development of
+PlasmaPy and affiliated packages.
+
+`PlasmaPy's online documentation`_ is hosted by `Read the Docs`_ and is
+available at these locations.
+
+* The documentation corresponding to the most recent official release
+  is labelled ``stable`` and is found at
+  `https://docs.plasmapy.org/ <https://docs.plasmapy.org/>`_ or
+  `https://docs.plasmapy.org/en/stable/ <https://docs.plasmapy.org/en/stable/>`_.
+
+* The documentation corresponding to the ``main`` branch on
+  `PlasmaPy's GitHub repository`_ is labelled ``latest`` and can be
+  found at `https://docs.plasmapy.org/en/latest/
+  <https://docs.plasmapy.org/en/latest/>`_.
+
+A preview of the documentation is generated every time a pull request
+is created or updated. You can access this preview by scrolling down
+to the checks at the bottom of a pull request, and clicking on
+``Details`` next to ``docs/readthedocs.org:plasmapy``.
 
 Building documentation
 ======================
-Documentation is built from the main branch on every commit pushed
-to it.
 
-Sphinx, the documentation generator of PlasmaPy, uses reStructuredText (reST) as its markup language. A primer on reST is available at this `webpage
+To install all dependencies required to develop PlasmaPy on your local
+computer, enter the top-level directory of the cloned repository and
+run
+
+.. code-block:: bash
+
+   pip install tox -r requirements.txt
+
+You can use `tox`_ to build the documentation from within the main
```

Comment:
A little foreshadowing here would be a could thing.  You should specifically say there are two method to build the documentation: using `tox` and using `make`.  Then maybe give a brief explanation why you would use one over the other.

Reply:
Could you say a bit more about the reasons to use `tox` vs. `make`?  I've mostly used `tox` since `make` brings back bad memories of trying to compile Fortran code.

Reply:
The big difference, `make` will just look at the current directory structure and build the documentation off of that.  This means you have to have properly setup all your dependencies ahead of time in whatever environment you are working in.  `tox` is a lot more computationally intensive since it will build the package and environment before building the documentation.  `tox` is great for CIs but I absolutely hate it for local builds because it does so much un-needed extra bits.